tests/test_scripts.py

```
@@ -55,12 +65,59 @@ def _run_script(script_name: str, csv_dir: str, output_dir: str) -> subprocess.C
 
 
 def test_plot_precision_generates_output(csv_dir):
+    """Legacy invocation: positional csv_dir + --output."""
     with tempfile.TemporaryDirectory() as outdir:
         result = _run_script("plot_precision.py", csv_dir, outdir)
         assert result.returncode == 0, f"Script failed: {result.stderr}"
+        # All five figures × two formats.
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)